POS/UserInterface/02b_Stores.md

```diff
@@ -275,69 +257,69 @@ For each pay desk, a tab with the printer settings below is displayed.
 - [CANCEL]   
     Click this button to cancel defining the stock settings and close the wizard window.
 
-    > [Info] If you cancel the wizard now, the store is nevertheless created but you have to configure the further store settings manually.
+    > [Info] If you cancel the wizard now, the store is nevertheless created, but you have to configure the further store settings manually.
 
 - [CONTINUE]   
     Click this button to save the stock settings and proceed to the next step in the store wizard. The *Saving...* window is displayed. When the saving is completed, the *Inventory assignment* wizard window is displayed.
 
 
-### Step 6 - Inventory assignment
+### Step 6 &ndash; Stock level assignment
 
 ![Create Store](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW06a.png "[Create Store]")
 
 - ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Create new inventory source*   
-    Activate this toggle to add a new inventory source that only considers the stock of the warehouse selected in the previous step for calculating the warehouse stock. Deactivate this toggle to select an existing inventory source in the drop-down list *Select existing inventory Source* or to add no inventory source. When you activate this toggle, the drop-down list *Select existing inventory Source* is hidden.
+    Activate this toggle to add a new stock source that only considers the stock level of the warehouse selected in the previous step for calculating the warehouse stock level. Deactivate this toggle to select an existing stock level source in the *Select existing inventory source* drop-down list, or to add no inventory source. This drop-down list is only displayed when the *Create new inventory source* toggle is disabled.  
```

Comment:
Zur Zeit würde ich bei "Stock" bleiben. Wir müssen wirklich klären, was mit stock source gemeint ist. Ich denke, siehe Screenshot unter Step 5 - Stock, dass man sich auf stock allocation bezieht (Im Text im Screenshot: "if you want that feature enable toggle "Configure stock source matrix"). Der Satz "Activate this toggle to add a new stock source that only considers the stock level of the warehouse selected in the previous step for calculating the warehouse stock level." wäre meiner Meinung nach: "considers the stock level (also, reales Bestand) for calculating the warehouse stock value (calculated)". "Stock level source" macht für mich wenig Sinn, daher würde ich momentan bei stock source bleiben und klären.